sklearn/linear_model/_base.py

```diff
@@ -687,6 +687,11 @@ def rmatvec(b):
         self._set_intercept(X_offset, y_offset, X_scale)
         return self
 
+    def __sklearn_tags__(self):
```

Comment:
I think that those could be moved in the `LinearModelCV` because `fit` and the `validate_data` is defined there.

Reply:
It should be done for all models inheriting from `LinearModelCV`.

Reply:
But `LinearRegression` inherits from `LinearModel` and not `LinearModelCV` ?

Reply:
Oups, I might have commented on the wrong line. I recall to have seen it for `ElasticNetCV` and `LassoCV` if I recall properly.